support-frontend/assets/pages/showcase/components/regionalContent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export const regionalContent = {
+  ukContent: {
```

Comment:
Use `CountryGroupId` constants as keys to this object

Reply:
Done